File: app/src/main/java/com/example/appbank/Data/Remote/ILoginService.java
package com.example.appbank.data.remote;

import com.example.appbank.data.remote.model.LoginRequest;
import com.example.appbank.data.remote.model.LoginResponse;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface ILoginService {
    @POST("login")
    Call<LoginResponse> postLogin(@Body LoginRequest loginRequest);
}

File: BankApp/app/src/main/java/br/com/rphmelo/bankapp/api/ILoginService.kt
package br.com.rphmelo.bankapp.api

import br.com.rphmelo.bankapp.models.LoginRequest
import br.com.rphmelo.bankapp.models.LoginResponse
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface ILoginService {

    @POST("login")
    fun login(@Body request: LoginRequest): Call<LoginResponse>
}